Objective-C/Internal/CBLCookie.h

```diff
@@ -0,0 +1,30 @@
+//
+//  CBLCookie.h
+//  CouchbaseLite
+//
+//  Copyright (c) 2020 Couchbase, Inc All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import <Foundation/Foundation.h>
+#import "CBLReplicator.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface CBLCookie : NSObject
```

Comment:
Can you note that this is for test to get the cookies?

Reply:
We don't have any unit test which tests this. so will go ahead and remove this. since this doesn't add much value.

Reply:
I used it to validate a unit test which used proxy app to inject cookie.